src/contracts/libraries/GPv2SafeERC20.sol

```diff
@@ -15,8 +15,16 @@ library GPv2SafeERC20 {
         address to,
         uint256 value
     ) internal {
-        bytes4 selector_ = token.transfer.selector;
+        bytes4 selector_ = IERC20.transfer.selector;
 
+        // NOTE: Perform an assembly transfer call and inspect the return data.
+        // Note that we write the return data to memory 0 in the case where the
+        // return data size is 32, this is OK since the first 64 bytes of memory
+        // are reserved by Solidy as a scratch space that can be used within
+        // assembly blocks.
+        // <https://docs.soliditylang.org/en/v0.7.6/internals/layout_in_memory.html>
+        bool badReturnSize;
```

Comment:
Where do we write to `badReturnSize`?

Reply:
https://github.com/gnosis/gp-v2-contracts/blob/7626419f9eef066f8ff667e5b8b9c3178d7ead81/src/contracts/libraries/GPv2SafeERC20.sol#L61

Its unfortunate that the diff just cuts it out. Essentially the change here is that we had a `getTransferResult` method as it was called in two separate places, but now its all part of the `transfer` function.